src/web/mod.rs

```diff
@@ -581,8 +582,8 @@ mod test {
             let release_id = release("0.3.0", db);
             let query = "UPDATE releases SET yanked = true WHERE id = $1 AND version = '0.3.0'";
             db.conn().query(query, &[&release_id]).unwrap();
-            assert_eq!(version(None, db), strip_id(MatchVersion::None));
-            assert_eq!(version(Some("0.3"), db), strip_id(MatchVersion::None));
+            assert_eq!(version(None, db), MatchVersion::None.strip_id());
+            assert_eq!(version(Some("0.3"), db), MatchVersion::None.strip_id());
```

Comment:
Do you need to call `strip_id()` here if the version is already `None`?

Reply:
hmm, yes, because they're different types. `version(Some("0.3"))` is `Option<String>` while the other is `MatchVersion`. Here `strip_id` is just used to convert into the correct type.

Reply:
Ok, it might be easier to just say `None` instead of `MatchVersion::None` then.